src/components/itemsComponents/BestProducts.tsx

```diff
@@ -0,0 +1,17 @@
+import { GetProductsResponse } from "@/types/products.type";
+import ProductCard from "./ProductCard";
+
+function BestProductList({
+  products: { list },
+}: {
+  products: GetProductsResponse;
+}) {
+  return (
+    <div>
+      {list.map((item, index) => (
+        <ProductCard item={item} key={item.id} index={index} />
+      ))}
+    </div>
```

Comment:
semantic
```suggestion
  return (
    <ul>
      {list.map((item, index) => (
        <ProductCard item={item} key={item.id} index={index} />
      ))}
    </ul>
```
시멘틱 태그를 준수하기 위해 ul태그로 추후 바꿔주세요:)